gtests/net/mptcp/mp_join/mp_join_server_bad_token.pkt

```diff
@@ -0,0 +1,15 @@
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
++0    `../common/server.sh`
+
++0.0  socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0  setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
++0.0  fcntl(3, F_GETFL) = 0x2 (flags O_RDWR)
++0.0  fcntl(3, F_SETFL, O_RDWR|O_NONBLOCK) = 0
+
++0    bind(3, ..., ...) = 0
++0    listen(3, 1) = 0
+
++0.0 < addr[caddr0] > addr[saddr0] S 0:0(0) win 65535 <mss 1460,mp_join_syn address_id=0 token=1>
```

Comment:
I don't remember, we always need `addr[x] > addr[y]` because of we ask for an MPJ? (it doesn't hurt, but maybe better without except if it is mandatory of course :) )

Reply:
Nope, I removed it, thanks!